### DISCUSSION TITLE
Not able to find NRF configuration script in wiki

### DISCUSSION THREAD

**Discussion by Deblina96** [OAI/OpenAPI-Specification#3490] [General]
I am following the below repo to install NRF. I have successfully build NRF. But after building I am unable to find nrf_conf.sh script to generate nrf.conf.

https://gitlab.eurecom.fr/oai/cn5g/oai-cn5g-nrf/-/wikis/Installation
Please, let me know where nrf_conf.sh exist.

**Comment by handrews** [comment]
You should file this issue on that GitLab repository for OAI (Open Air Interface) NRF.  This repository is for OpenAPI.  I do not think these things are related.
